server/database/fake-data/fakeUsers.json

```diff
@@ -0,0 +1,116 @@
+[
```

Comment:
Nice list to have ,but might be hard to authenticate those users since their passwords are not encrypted with bcrypt. FYI